I tried cherry-picking this in on top of current master, but `iproute2` in the closure fails to build: 

```
   48 | #define IFF_VOLATILE (IFF_LOOPBACK|IFF_POINTOPOINT|IFF_BROADCAST| \
      | 
    CC       ipmptcp.o
    CC       iplink_bareudp.o
    CC       iplink_wwan.o
    CC       ipioam6.o
    CC       iplink_amt.o
    CC       iplink_batadv.o
    CC       iplink_gtp.o
    CC       iplink_virt_wifi.o
    CC       ipstats.o
    CC       rtmon.o
ipstats.c: In function ‘ipstats_show_64’:
ipstats.c:143:48: warning: implicit declaration of function ‘MIN’ [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-function-declaration-Wimplicit-function-declaration8;;]
  143 |                 memcpy(__dest, RTA_DATA(__at), MIN(__at_sz, __var_sz)); \
      |                                                ^~~
ipstats.c:157:9: note: in expansion of macro ‘IPSTATS_RTA_PAYLOAD’
  157 |         IPSTATS_RTA_PAYLOAD(stats, at);
      |         ^~~~~~~~~~~~~~~~~~~
    LINK     rtmon
    LINK     ip
/nix/store/jvsyjspqyrj6h6585d66k6xg3dyap5zc-binutils-2.38/bin/ld: ipstats.o: in function `ipstats_show_64':
ipstats.c:(.text+0x245): undefined reference to `MIN'
/nix/store/jvsyjspqyrj6h6585d66k6xg3dyap5zc-binutils-2.38/bin/ld: ipstats.o: in function `ipstats_stat_desc_show_l3_stats':
ipstats.c:(.text+0xe02): undefined reference to `MIN'
/nix/store/jvsyjspqyrj6h6585d66k6xg3dyap5zc-binutils-2.38/bin/ld: ipstats.o: in function `ipstats_stat_desc_show_afstats_mpls':
ipstats.c:(.text+0x1bc0): undefined reference to `MIN'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:29: ip] Error 1
make: *** [Makefile:76: all] Error 2
error: builder for '/nix/store/m2ax47lgn56khml5bc2gaw4rgzp8mqrz-iproute2-5.19.0.drv' failed with exit code 2
error: 1 dependencies of derivation '/nix/store/8sa4nc2sa278nld6sdfchqphg4nqwglc-tpm2-tss-3.2.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/qk61n3ddi5xd7lz0c6v9krzv49b5fbfl-systemd-251.4.drv' failed to build
```

Is this fixed on staging?

--- a/pkgs/os-specific/linux/systemd/default.nix
+++ b/pkgs/os-specific/linux/systemd/default.nix
@@ -98,8 +98,8 @@
 , withShellCompletions ? true
 , withTimedated ? true
 , withTimesyncd ? true
-, withTpm2Tss ? !stdenv.hostPlatform.isMusl
-, withUserDb ? !stdenv.hostPlatform.isMusl
+, withTpm2Tss ? true
+, withUserDb ? true
 , withUtmp ? !stdenv.hostPlatform.isMusl
   # tests assume too much system access for them to be feasible for us right now
 , withTests ? false

